### PR TITLE
creation of contact us page

### PR DESCRIPTION
**Pull request by himasheth**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "node-sass": "^4.14.1",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-native": "^0.63.4",
```
Comment (peiyaaao) [inline package.json:20]: Do we need this dependency? React native is used for mobile development.

File: src/components/about/About.tsx [hunk 2]
```diff
@@ -1,5 +1,105 @@
...
+import placeholder2 from '../../images/box2.svg';
+import girlLaptop from '../../images/girl-with-laptop.svg';
+import dudeSitting from '../../images/sitting-down-dude.svg';
+import meetingPpl from '../../images/meeting-ppl.svg';
+import artSesh from '../../images/art-sesh.svg';
 
 export default function About() {
-  return <div>About Us</div>;
+  return (
+    <div className="parent">
```
Comment (peiyaaao) [inline src/components/about/About.tsx:14]: Should the about us sections be in here?

File: src/components/about/Process.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import PropTypes from 'prop-types';
+import React from 'react'; // we need this to make JSX compile
+
+type CardProps = {
+  title: string;
+  image: any;
+  description: string;
+};
+
+export const Card = ({ title, image, description }: CardProps) => (
```
Comment (peiyaaao) [inline src/components/about/Process.tsx:11]: Why do we have both a `Card` and `Process`?

File: src/components/about/ProcessList.tsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+import placeholder from '../../images/box1.svg';
+import processplaceholder from '../../images/process-placeholder.svg';
+
+type CardProps = {
+  title: string;
+  image: any;
+  description: string;
+};
+
+export const Card = ({ title, image, description }: CardProps) => (
```
Comment (peiyaaao) [inline src/components/about/ProcessList.tsx:14]: Why do we have both a `Card` and `Process`?

File: src/components/about/Values.tsx [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+import PropTypes from 'prop-types';
+import React from 'react'; // we need this to make JSX compile
+
+type CardProps = {
+  title: string;
+  image: any;
+  description: string;
+};
+
+export const Card = ({ title, image, description }: CardProps) => (
```
Comment (peiyaaao) [inline src/components/about/Values.tsx:11]: Same, here do we need both `Card` and `Values`

File: src/components/contact/Contact.scss [hunk 6]
```diff
@@ -0,0 +1,90 @@
+.contact-us {
```
Comment (peiyaaao) [inline src/components/contact/Contact.scss:1]: For the styling can we follow https://medium.com/@andrew_barnes/bem-and-sass-a-perfect-match-5e48d9bc3894. Also I believe the navbar.scss file also follows this style.

File: src/images/girl-with-laptop.svg [hunk 7]
```diff
@@ -0,0 +1,28 @@
+<svg width="251" height="225" viewBox="0 0 251 225" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M59.1223 130.446C20.1991 79.5725 -13.6682 9.7807 74.7535 39.2754C163.175 68.7701 191.028 -56.5415 227.389 137.474C249.827 230.426 98.0454 181.319 59.1223 130.446Z" fill="#E0F0FF"/>
```
Comment (peiyaaao) [inline src/images/girl-with-laptop.svg:2]: Could we rename the svg files to follow the naming style you used here for this svg file

File: package.json (hunk 1 above)
Comment (himasheth) [reply package.json:20]: I guess not I think it just automatically downloaded when I installed react haha

File: src/components/about/About.tsx (hunk 2 above)
Comment (himasheth) [reply src/components/about/About.tsx:14]: Should I just remove all the code in the file altogether?

File: src/components/about/Process.tsx (hunk 3 above)
Comment (himasheth) [reply src/components/about/Process.tsx:11]: Not sure, I think I pulled our about-us branch when we were doing the scaffolding so I accidentally got our old work in here. Should I just empty the file?

File: src/components/about/ProcessList.tsx (hunk 4 above)
Comment (himasheth) [reply src/components/about/ProcessList.tsx:14]: See above.

File: src/components/contact/Contact.scss (hunk 6 above)
Comment (himasheth) [reply src/components/contact/Contact.scss:1]: Yep I can do that!

File: src/images/girl-with-laptop.svg (hunk 7 above)
Comment (himasheth) [reply src/images/girl-with-laptop.svg:2]: sure, just lowercase letters and dashes?

**Comment by himasheth** [comment]
I'm still working on the responsiveness and the image positioning might need help with that but other than that it should be good. The images are the only things that get super messy when on a phone.

File: src/images/girl-with-laptop.svg (hunk 7 above)
Comment (peiyaaao) [reply src/images/girl-with-laptop.svg:2]: yep

File: src/components/about/Process.tsx (hunk 3 above)
Comment (peiyaaao) [reply src/components/about/Process.tsx:11]: Yep, delete anything that isn't explicitly related to your contact us page.

**Pull request closed without merging**